nomenclature/definitions/variable/technology/technologies.yaml

```
@@ -3,263 +3,98 @@
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Coal|w/ CCS:
-   description: Capital cost of a new coal power plant with CCS
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Coal|w/o CCS:
-   description: Capital cost of a new coal power plant w/o CCS
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Gas|w/ CCS:
-   description: Capital cost of a new gas power plant with CCS
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Gas|w/o CCS:
-   description: Capital cost of a new gas power plant w/o CCS
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Geothermal:
-   description: Capital cost of a new geothermal power plant
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Hydro:
-   description: Capital cost of a new hydropower plant
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Nuclear:
-   description: Capital cost of a new nuclear power plants
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Solar|CSP:
-   description: Capital cost of a new concentrated solar power plant
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Solar|PV:
-   description: Capital cost of a new solar PV units
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Wind|Offshore:
-   description: Capital cost of a new offshore wind power plants
-   unit: US$2010/kW
-
-Capital Cost|Electricity|Wind|Onshore:
-   description: Capital cost of a new onshore wind power plants
-   unit: US$2010/kW
-
-Capital Cost|Gases|Biomass|w/o CCS:
-   description: Capital cost of a new biomass to gas plant w/o CCS
-   unit: US$2010/kW
-
-Capital Cost|Gases|Coal|w/o CCS:
-   description: Capital cost of a new coal to gas plant w/o CCS
-   unit: US$2010/kW
-
-Capital Cost|Hydrogen|Biomass|w/ CCS:
-   description: Capital cost of a new biomass to hydrogen plant with CCS
-   unit: US$2010/kW
-
-Capital Cost|Hydrogen|Biomass|w/o CCS:
-   description: Capital cost of a new biomass to hydrogen plant w/o CCS
-   unit: US$2010/kW
-
-Capital Cost|Hydrogen|Coal|w/ CCS:
-   description: Capital cost of a new coal to hydrogen plant with CCS
-   unit: US$2010/kW
-
-Capital Cost|Hydrogen|Coal|w/o CCS:
-   description: Capital cost of a new coal to hydrogen plant w/o CCS
-   unit: US$2010/kW
-
-Capital Cost|Hydrogen|Electricity:
-   description: Capital cost of a new hydrogen-by-electrolysis plant
-   unit: US$2010/kW
-
-Capital Cost|Hydrogen|Gas|w/ CCS:
-   description: Capital cost of a new gas to hydrogen plant with CCS
-   unit: US$2010/kW
-
-Capital Cost|Hydrogen|Gas|w/o CCS:
-   description: Capital cost of a new gas to hydrogen plant w/o CCS
-   unit: US$2010/kW
-
-Capital Cost|Liquids|Biomass|w/ CCS:
-   description: Capital cost of a new biomass to liquids plant with CCS
-   unit: US$2010/kW
+Capacity|Transportation|Freight:
+   description: Total installed freight transportation technologies
+   unit: Gtkm
+   
+Capacity|Transportation|Freight|<Transport>:
+   description: Total installed freight transportation technologies from <this transport>
```

Suggest to rename the exploding tag for readability.
```suggestion
   description: Total installed freight transportation technologies from <this transport mode>
```